app/TerrainRendererItem.cpp

```diff
@@ -278,6 +278,20 @@ void TerrainRendererItem::set_camera_operation_centre_visibility(bool new_camera
     emit camera_operation_centre_visibility_changed();
 }
 
+float TerrainRendererItem::camera_operation_centre_distance() const
+{
+    return m_camera_operation_centre_distance;
+}
+
+void TerrainRendererItem::set_camera_operation_centre_distance(float new_camera_operation_centre_distance)
+{
+    if (m_camera_operation_centre_distance == new_camera_operation_centre_distance)
+        return;
+    qDebug() << new_camera_operation_centre_distance;
```

Comment:
please remove the debug outpu

Reply:
removed the debug output.

it would be easy to add it to orbit but i'm not sure if it makes sense as the point is always on the surface where estimating distance is way easier and the bigger point would just hide information.

switching between modes works and is tested.